OnlineLibrary.API/Controllers/BookController.cs

```diff
@@ -31,6 +32,15 @@ public BookController(IBookService iBook, IAuthorService iAuthor, IMapper mapper
             _mapper = mapper;
         }
 
+        // GET: api/Book/GetAllBooks/
+        [HttpGet("{PageNumber}/{PageSize}")]
+        public IActionResult GetAllBooks(int? PageNumber, int? PageSize)
```

Comment:
Make it [HttpPost("search")]
Create model SearchQuery or pass PaginationOptions directly with int PageNumber, int PageSize